### ISSUE TITLE
useGatsbyConfig requires .ts file extension

### ISSUE THREAD

**Issue by orsi** [Js-Brecht/gatsby-plugin-ts-config#40]
As per the readme, I was under the assumption I could just use the filename without .ts ending:

```
// gatsby-config.js
const { useGatsbyConfig } = require('gatsby-plugin-ts-config');
module.exports = useGatsbyConfig('./gatsby-config');
```

However, I get errors relating specifically to the gatsby-theme-i18n plugin:

<img width="568" alt="Screen Shot 2021-09-03 at 11 45 50 AM" src="https://user-images.githubusercontent.com/7156655/132032617-1fadfc99-b98d-4ca2-9190-ebe67b27b0a1.png">

Those errors are resolved when I specify .ts explicitly:

```
// gatsby-config.js
const { useGatsbyConfig } = require('gatsby-plugin-ts-config');
module.exports = useGatsbyConfig('./gatsby-config.ts');
```

No errors:
<img width="566" alt="Screen Shot 2021-09-03 at 11 47 58 AM" src="https://user-images.githubusercontent.com/7156655/132032895-2fa5fdbf-7e83-4e2b-888e-4eb782c8d39d.png">

**Comment by Js-Brecht** [comment]
What version are you using?  That was fixed in `2.0.1`

**Comment by orsi** [comment]
Just recently installed it at 2.0.1:

![image](https://user-images.githubusercontent.com/7156655/132044573-4ca8c331-cff1-4ae4-83b3-1f7ada504e66.png)

**Comment by Js-Brecht** [comment]
Ah, you are putting `gatsby-config.ts` right next to `gatsby-config.ts`.  It’s finding the `.js` file, because node automatically prioritizes them.

I suppose in the transpiler I could use the other module resolver, instead of `require.resolve`… 

I’ve also got a feature nearly done that will include a CLI, so no `.js` files needed at all.

**Comment by orsi** [comment]
Ahh, yes. Kind of thought that might be the case which is why I added the .ts.

**Comment by Js-Brecht** [comment]
@orsi care to try `gatsby-plugin-ts-config@2.0.2-rc.1`?

**Comment by Js-Brecht** [comment]
Did some tests, seems to work as expected.  Published in `gatsby-plugin-ts-config@2.0.2`

**Comment by orsi** [comment]
Updated and works on my end now without the explicit .ts extension. Thanks!
